AwsEncryptionSDK/runtimes/net/ESDK.csproj

```diff
@@ -29,10 +29,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-      <PackageReference Include="AWSSDK.Core" Version="3.7.103" />
+      <!-- TODO: manually upgraded to match the latest from MPL, is that reasonable? -->
+      <PackageReference Include="AWSSDK.Core" Version="3.7.300.2" />
       <PackageReference Include="DafnyRuntime" Version="4.2.0" />
```

Comment:
Don't you need to update the runtime for this to work?

Reply:
In general yes, looks like I got away with not handling this here yet since there weren't any relevant changes in the .NET runtime, unlike the Java one. Since the PR is already pretty big can we leave that for a future PR, at worst when this repo actually upgrades to a new Dafny?

Reply:
ok then.